invenio_app_ils/mail/tasks.py

```diff
@@ -30,7 +40,13 @@ def send_ils_email(message, name="ils_mail"):
     log_msg = dict(
         name=name, action="before_send", message_id=dump["id"], data=dump
     )
-    celery_logger.debug(json.dumps(log_msg, sort_keys=True))
+    celery_logger.info(json.dumps(log_msg, sort_keys=True))
+
+    if called_by_celery_task():
+        dump["triggered"] = "Automatically"
+    else:
+        dump["triggered"] = "Manually"
```

Comment:
Can we have an ENUM for this (not in the db field, but just a string), probably in the model class?